src/main/java/org/folio/innreach/domain/entity/InnReachTransaction.java

```diff
@@ -87,6 +93,13 @@
   public static final String FETCH_OPEN_BY_ITEM_AND_PATRON_QUERY = FETCH_OPEN_TRANSACTIONS +
     " AND h.folioItemId = :folioItemId AND h.folioPatronId = :folioPatronId";
 
+
+  public static final String FETCH_ASSOCIATED_QUERY_NAME = "InnReachTransaction.fetchAssociatedLoan";
+  public static final String FETCH_ASSOCIATED_QUERY = "SELECT irt FROM InnReachTransaction AS irt " +
```

Comment:
what does "fetch associated query" mean? Update the name to describe the query similar to existing ones

Reply:
Done